app/src/main/res/values-de/strings.xml

```diff
@@ -64,52 +64,52 @@
     <string name="disk_header_size">Größe</string>
     <string name="disk_header">Speichernutzung</string>
     <string name="disk_header_avail">Frei</string>
-    <string name="disk_header_usedper">Belegt%</string>
+    <string name="disk_header_usedper">Belegt</string>

```

Comment:
Größe und Frei sind in Bytes angegeben, Belegt dazu in Prozent. Sollte man auch an der Überschrift sehen.